meinberlin/react/kiezradar/SearchProfiles.jsx

```diff
@@ -0,0 +1,294 @@
+  }
+
+  if (error) {
+    return (
+      <div className="search-profiles-list__error">
+        {errorText}: {error}
+      </div>
+    )
+  }
+
+  return (
+    <>
+      {alert && <Alert onClose={() => setAlert(false)} />}
+      {searchProfiles.length === 0
+        ? (
+          <>
+            <h2 className="search-profiles-list__title">{noSavedProfilesText}</h2>
+            <a href={planListUrl} className="button button--light">
+              <i className="fa-solid fa-magnifying-glass mr-1" />
+              {findProjectsText}
+            </a>
+          </>
+          )
+        : (
+          <>
+            <h2 className="search-profiles-list__title">
+              {yourSavedProfilesText} ({searchProfiles.length})
+            </h2>
+            {searchProfiles.map((profile) => (
+              <SearchProfile
+                key={profile.id}
+                apiUrl={apiUrl}
+                planListUrl={planListUrl}
+                profile={profile}
+                onDelete={(id) => {
+                  setAlert(true)
+                  setSearchProfiles((prevSearchProfiles) =>
+                    prevSearchProfiles.filter((profile) => profile.id !== id)
+                  )
+                }}
+              />
+            ))}
+          </>
+          )}
+    </>
+  )
+}
+
+function SearchProfile ({ apiUrl, planListUrl, profile: profile_, onDelete }) {
+  const [isEditing, setIsEditing] = useState(false)
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState(null)
+  const [profile, setProfile] = useState(profile_)
+
+  const handleDelete = async () => {
+    setLoading(true)
+    setError(null)
+
+    try {
+      const response = await fetch(apiUrl + profile.id + '/', {
+        headers: {
+          'X-CSRFToken': cookie.get('csrftoken')
+        },
+        method: 'DELETE'
+      })
+
+      if (!response.ok) {
+        throw new Error(errorDeleteSearchProfilesText)
+      }
+
+      onDelete(profile.id)
+    } catch (err) {
+      setError(err.message)
+    } finally {
+      setLoading(false)
+    }
+  }
+
+  const handleSubmit = async (e) => {
+    e.preventDefault()
+    setLoading(true)
+    setError(null)
+
+    try {
+      const response = await fetch(apiUrl + profile.id + '/', {
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8',
+          'X-CSRFToken': cookie.get('csrftoken')
+        },
+        method: 'PATCH',
+        body: JSON.stringify({ name: e.target.elements.name.value })
+      })
+
+      if (!response.ok) {
+        throw new Error(errorUpdateSearchProfilesText)
+      }
+
+      const data = await response.json()
+      setProfile(data)
+    } catch (err) {
+      setError(err.message)
+    } finally {
+      setIsEditing(false)
+      setLoading(false)
+    }
+  }
+
+  return (
+    <div className="search-profile">
+      <div className="search-profile__header">
+        <h3 className="search-profile__title">{profile.name}</h3>
```

Comment:
It should also show what filters have been set, no?
![CleanShot 2025-01-07 at 07 29 19@2x](https://github.com/user-attachments/assets/61ca59d5-3be4-4588-8cda-db2e1fc2446f)